how-to-make-a-release

```diff
@@ -0,0 +1,4 @@
+1) Update VERSION file and commit
+2) Tag new release with "v${VERSION}"
+3) Run "/scripts/git-archive-all.sh --format tar --prefix clr-boot-manager-${VERSION}/ -t v${VERSION} clr-boot-manager-${VERSION}.tar && xz clr-boot-manager-${VERSION}.tar"
```

Comment:
```suggestion
3) Run "./scripts/git-archive-all.sh --format tar --prefix clr-boot-manager-${VERSION}/ -t v${VERSION} clr-boot-manager-${VERSION}.tar && xz clr-boot-manager-${VERSION}.tar"
```

Reply:
Or does `./scripts/mkrelease.sh` just need to be tweaked a bit to accomplish all of these steps instead?

Reply:
mkrelease.sh will need to be poked a little. I don't use it since I don't sign the archives.

Reply:
Updated